### PR TITLE
CT-1240 test bucket drop

### PR DESCRIPTION
**Pull request by jirkasemmler**
Jira: CT-1240
KBC: -

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)
---

just add dropBucket step in the flow to check that bucket drop really works in BYODB env

### CODE REVIEW

File: tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php [hunk 1]
```diff
@@ -80,21 +91,33 @@ public function testRegisterExternalBucketInBYODBEnvironment(): void
...
+            ),
+        );
+
+        $this->_client->refreshBucket($bucketId);
+
+        $tables = $this->_client->listTables($bucketId);
         $this->assertCount(1, $tables);
-        $this->_client->refreshBucket('in.test-bucket-registration');
 
+        $this->_client->dropBucket($bucketId);
```
Comment (jirkasemmler) [inline tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php:116]: todle je realna zmena, zbytek je omacka, co jsem si neodpustil, kdyz jsem to videl

File: tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php [hunk 2]
```diff
@@ -80,21 +91,33 @@ public function testRegisterExternalBucketInBYODBEnvironment(): void
...
+        $this->assertCount(2, $tables);
+
+        $db->executeQuery(
+            sprintf(
+                'DROP VIEW %s;',
+                SnowflakeQuote::quoteSingleIdentifier(self::TEST_VIEW),
+            ),
+        );
+
+        $this->_client->refreshBucket($bucketId);
```
Comment (jirkasemmler) [inline tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php:111]: chtel jsem checknout, ze ten refresh realne neco udela

**Comment by romanbracinik** [comment]
👁️

File: tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php (hunk 1 above)
Comment (romanbracinik) [reply tests/Backend/ExternalBuckets/SnowflakeBYODBTest.php:116]: Toto tu asi nemusi byt, maze sa to na zaciatku testu, ale nicomu to nevadi

**Review by romanbracinik — APPROVED** [review]
lgtm

**Pull request merged**